web/client/components/contextcreator/__tests__/ContextCreator-test.jsx

```diff
@@ -26,4 +28,30 @@ describe('ContextCreator component', () => {
         const container = document.getElementById('container');
         expect(container.getElementsByClassName('ms2-stepper')[0]).toExist();
     });
+    describe('Test ContextCreator onSave', () => {
+        it('default destination', () => {
+            const actions = {
+                onSave: () => { }
+            };
+            const spyonSave = expect.spyOn(actions, 'onSave');
+            ReactDOM.render(<ContextCreator onSave={actions.onSave} />, document.getElementById("container"));
+            // save button
+            const button = document.querySelectorAll('.footer-button-toolbar-div button')[1];
```

Comment:
this is fine, but naming button => saveButton is self explanatory withouth the need of the comment
```suggestion
            const saveBtn = document.querySelectorAll('.footer-button-toolbar-div button')[1];
```